content/user/apps/TrackLibraries.md

```diff
@@ -32,22 +32,36 @@ Export details on findings from the Libraries page, a library's overview page, o
 
 <a href="assets/images/Library-export.png" rel="lightbox" title="Export library details"><img class="thumbnail" src="assets/images/Library-export.png"/></a>
 
-In the Libraries grids, select individual libraries with the check boxes or use your filters to focus on specific data sets. When you apply a selection, you can choose to export all or selected libraries. If you don't select any checkboxes or filters, you can export data for all libraries. 
+In the Libraries grids, select individual libraries or use the filters to focus on specific data sets. If you apply a selection, you can choose to export all or selected libraries; if not, you can export data for all libraries. 
```

Comment:
@kaitlynmortimer This second sentence just makes things sound confusing and complicated to me. How about "You may choose to either export all libraries or just those you've selected." I think the difference between what they can do from that icon with or without selections is easily discoverable in the UI and doesn't need to be so explicitly outlined.